example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -86,24 +90,19 @@ describe('The S3 Ingest Granules workflow', () => {
       await s3().deleteObject({ Bucket: files[3].bucket, Key: files[3].filepath }).promise();
     });
 
-    it('has a payload with updated filename', () => {
-      let i;
-      for (i = 0; i < 4; i += 1) {
-        expect(files[i].filename).toEqual(expectedPayload.granules[0].files[i].filename);
-      }
+    it('has a payload with correct buckets and filenames', () => {
+      files.forEach((file) => {
+        const expectedFile = expectedPayload.granules[0].files.find((f) => f.name === file.name);
+        expect(file.filename).toEqual(expectedFile.filename);
+        expect(file.bucket).toEqual(expectedFile.bucket);
+      });
     });
 
     it('moves files to the bucket folder based on metadata', () => {
       existCheck.forEach((check) => {
         expect(check).toEqual(true);
       });
     });
-
```

Comment:
Did you get rid of this test?

Reply:
I guess it's handled now when the output is compared?

Reply:
Correct.